samples/client/petstore/graphql-client/petstore/api/pet_api.graphql

```diff
@@ -0,0 +1,60 @@
+# OpenAPI Petstore
+# This is a sample server Petstore server. For this sample, you can use the api key `special-key` to test the authorization filters.
+# OpenAPI spec version: 1.0.0
+# Generated by OpenAPI Generator: https://openapi-generator.tech
+#
+
+# package petstore
+
+# pet_api
+
+type Query {
+  # Add a new pet to the store
+  # @param Pet $pet Pet object that needs to be added to the store
+  # @return [null]
+  AddPet($pet: Pet!): null
+
+  # Deletes a pet
+  # @param Int $petId Pet id to delete
+  # @param String $apiKey 
+  # @return [null]
+  DeletePet($petId: Int!, $apiKey: String): null
```

Comment:
also, I wonder why you are using "$petId" instead of "petId"? (same about other parameters)

Reply:
I read about variables in [GraphQL documentation](https://graphql.org/learn/queries/#variables)  and to me it makes sense to use variables, which supports default values (something we can add later)

Reply:
Those are about querying the graphql server, not the server definition
So we can name our parameters without dollar sign and we are able to send default values from frontend (also there is a way for having backend defaults)

Reply:
For this particular generator, I've named it as "graphql-client" as a GraphQL client only. We can create another generator for "graphql-server"

Reply:
Wow... I didn't know that
But there is no need for a graphql client at all... graphql is super declarative so a client is useless, also the graphql point is that we can have very customized queries and a client will annihilate graphql
also in the client, we don't have "type" keyword or any "Query & Mutation" types

Reply:
No problem. We can switch to graphsql-server and consider a graphql-client at later stage if needed.

I'll take your advice to remove $ from the parameter.